testcharms/charm-repo/bionic/dispatch/config.yaml

```diff
@@ -0,0 +1,5 @@
+options:
```

Comment:
Do we actually need this file?

Reply:
Not necessarily, but it does allow us to change config if we want.